packages/dapp-svelte-wallet/react-ui/src/components/WalletConnection.js

```diff
@@ -78,12 +94,20 @@ const WalletConnection = ({ setConnectionState }) => {
   }, []);
 
   return (
-    <span className="hidden">
-      <AgoricWalletConnection onState={onWalletState} />
-    </span>
+    <div className={classes.connector}>
```

Comment:
Great, this is exactly how I intended the wallet connection component to be used!  Hidden, with some other component exposing state.